tests/home.spec.js

```diff
@@ -49,3 +51,24 @@ test.describe("accessibility tests (dark)", () => {
     expect(accessibilityScanResults.violations).toEqual([]);
   });
 });
+
+test.describe("Data rendered correctly on home page", () => {
+  test("Total active rendered", async ({ page }) => {
+    const { stats: totalStats } = await getTotalStats();
+    const abbreviatedActive = abbreviateNumber(totalStats.active);
+    await page.goto("/");
+    const section = page.getByText("Active Users");
+    const parent = section.locator("..");
+    const item = parent.locator("dd div:nth-child(1)");
+    await expect(item).toHaveText(abbreviatedActive);
+  });
+
+  test("Profile views rendered", async ({ page }) => {
+    const { stats: totalStats } = await getTotalStats();
+    const abbreviatedViews = abbreviateNumber(totalStats.views);
+    await page.goto("/");
+    const viewsLocator = await page.locator("dl > div:nth-child(2) dd div:nth-child(2)");
+    const finalTotalViews = await viewsLocator.textContent();
+    expect(finalTotalViews).toEqual(abbreviatedViews);
```

Comment:
Same here for the selectors

```suggestion
    const section = page.getByText("Profile Views");
    const parent = section.locator("..");
    const item = parent.locator("dd div:nth-child(1)");
    await expect(item).toHaveText(abbreviatedViews);
```